.changes/next-release/bugfix-eks gettoken-28465.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We should remove this changelog entry since we already included it in a v2 release.